packages/sources/coingecko/src/endpoint/crypto.ts

```diff
@@ -36,7 +45,16 @@ export const endpointResultPaths: {
 }
 
 export const description =
-  '**NOTE: the `price` endpoint is temporarily still supported, however, is being deprecated. Please use the `crypto` endpoint instead.**'
+  '**NOTE: the `price` endpoint is temporarily still supported, however, is being deprecated. Please use the `crypto` endpoint instead.**\n' +
+  'Overrides can be provided as input parameters, as specified below, or be hardcoded into the `symbolToSymbolOverrides.json` or `symbolToIdOverrides.json` files in the `./src/config folder`.  Overrides specified as input parameters take precedence.  The order of operations for performing overrides is as follows: \n' +
```

Comment:
This seems pretty overkill here. A README in the Override class would be a better place for this.

Reply:
I removed the description here and moved it to the bootstrap/lib/modules README file.